### PR TITLE
Sd 2087

### PR DESCRIPTION
**Pull request by pritamstyz4ever**
## Context

API based responses need to be modified to match response schema for executor methods.

## Objective

This PR does a minor refactor to pass token and response object.

## References

https://github.com/screwdriver-cd/screwdriver/issues/2087

## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Comment by tkyi** [comment]
need to update jsdocs to match (since `config` is no longer a param)

File: index.js [hunk 1]
```diff
@@ -211,28 +203,28 @@ class ExecutorQueue extends Executor {
...
-            ...args
+            ...args,
+            body: config
         };
 
         return new Promise((resolve, reject) => {
             requestretry(options, (err, response) => {
                 if (!err) {
-                    if (response.statusCode === 200) {
-                        return resolve(response);
```
Comment (jithine) [inline index.js:231]: what changed ? don't we have this case anymore ?

Comment (pritamstyz4ever) [reply index.js:231]: We are returning everything, irrespective of statusCode

**Comment by pritamstyz4ever** [comment]
> need to update jsdocs to match (since `config` is no longer a param)

config is still a param

**Review by jithine — DISMISSED** [review]

**Review by jithine — APPROVED** [review]

**Review by parthasl — APPROVED** [review]
👍

**Pull request merged**